graphene/utils/str_converters.py

```diff
@@ -18,4 +18,4 @@ def to_snake_case(name):
 
 
 def to_const(string):
-    return re.sub('[\W|^]+', '_', string).upper()
+    return re.sub('[\W|^]+', '_', string.format()).upper()
```

Comment:
In Django, we would use [`force_text`](https://docs.djangoproject.com/en/1.9/ref/utils/#django.utils.encoding.force_text) to do something like this. I guess that's probably not an option for graphene itself, although it could be in the Django field transfer step. this might be a better fix.